spacy/tests/util.py

```diff
@@ -94,3 +94,24 @@ def assert_packed_msg_equal(b1, b2):
     for (k1, v1), (k2, v2) in zip(sorted(msg1.items()), sorted(msg2.items())):
         assert k1 == k2
         assert v1 == v2
+
+def assert_span_list_equal(spans_1, spans_2) :
+    spans_1 = list(spans_1)
+    spans_2 = list(spans_2)
```

Comment:
If these are not assumed to be lists on input, shouldn't they be sorted as well before zipping and comparing element by element?

Reply:
The order in lists is important when comparing. I overdid the type conversion there, for sure. I actually changed the function from asserting to simply comparing, to eliminate the need of a separate negative assertion helper.